src/API/Constants.ts

```diff
@@ -0,0 +1 @@
+export const redditData = 'https://www.reddit.com/r/technology/new.json'
```

Comment:
A constant name should represent exactly what it contains. `redditData` => `redditUrl` or something more appropriate.